convention-develocity-gradle-plugin/README.md

```diff
@@ -1,8 +1,44 @@
 ## Convention Develocity Gradle Plugin
 
-This project demonstrates how a convention plugin can share the same Develocity build configuration across multiple projects. It 
-is intended to serve as a starting point for creating your own Gradle plugin that applies your specific Develocity configuration. Note
-the inline comments in the build and source code for things to adjust specifically to your needs.
+This project demonstrates how a convention plugin can share the same Develocity build configuration across multiple projects.
+It is intended to serve as a starting point for creating your own Gradle plugin that applies your specific Develocity configuration.
+Note the inline comments in the build and source code for things to adjust specifically to your needs.
+
+### Applying the plugin to your build
+
+#### Using a dynamic version
+
+For builds using Gradle 6.9 and later, it's recommended to apply the convention plugin using a [dynamic version](https://docs.gradle.org/current/userguide/dynamic_versions.html).
+This approach ensures that the latest version of the plugin is automatically used, but only for the specified major version.
+For example, the following will automatically use the latest `1.x` version of the plugin up to, but excluding, version `2.0`:
+
+```groovy
+plugins {
+    id 'com.myorg.convention-develocity-gradle-5-or-newer' version '1.+'
+}
+```
+
+This allows you to quickly roll out non-breaking changes to all consumers of the convention plugin.
+Breaking changes in the plugin should be released under a new major version, e.g., `2.0`.
+All consumers of the plugin will then need to update the specified major version, e.g., `2.+`.
+
+> [!IMPORTANT]
+> Using dynamic versions should only be done when releases and development versions are published to separate repositories.
+> Not having this separation introduces the risk that consumers will use a development version of the plugin not yet ready to be used.
+> In these scenarios, using a static version is preferred.
+
+#### Using a static version
+
+For projects using earlier versions of Gradle, the convention plugin must be applied using a static version.
+For example:
+
+```groovy
+plugins {
+    id 'com.myorg.convention-develocity-gradle-5-or-newer' version '1.0'
+}
+```
+
+For each new release of the convention plugin, builds will need to be explicitly updated.
```

Comment:
`...consuming builds...`

Reply:
Updated for both Gradle and Maven in 264f19e65178bb018d07c60856a269fbf1ee7e65.